### PR TITLE
Mark oculus

### PR DESCRIPTION
**Pull request by mroukema**
Adjusted keyboard key text positioning

Made interaction panel functional and hooked it up to force field

### CODE REVIEW

File: Virtual Keyboard/Assets/Scripts/panelBehaviorScript.cs [hunk 1]
```diff
@@ -4,82 +4,60 @@
...
 
 	GameObject user;
-	GameObject forceField;
-	TextMesh panelText;
+	public GameObject forceField;
+	public string panelRequiredText;
+	public float interactionRange;
+	TextMesh promptTextMesh;
+	TextMesh inputTextMesh;
 	// var panelText;
```
Comment (eh-dub) [inline Virtual Keyboard/Assets/Scripts/panelBehaviorScript.cs]: delete this

File: Virtual Keyboard/Assets/Scripts/panelBehaviorScript.cs [hunk 2]
```diff
@@ -4,82 +4,60 @@
...
 
 	// Use this for initialization
 	void Start () {
 		user = GameObject.Find ("User");
```
Comment (eh-dub) [inline Virtual Keyboard/Assets/Scripts/panelBehaviorScript.cs]: Use method that does not require static String

**Comment by mroukema** [comment]
Branch updated based on feedback.

**Comment by eh-dub** [comment]
Tight.

**Pull request merged**